### PR TITLE
Message for OSError: [Errno2]

### PR DESCRIPTION
**Pull request by HassenPy**
#94 
This halts the script unless hostapd is installed giving a colored custom message.

### CODE REVIEW

**Comment by HassenPy** [comment]
Looks like this

File: wifiphisher.py [hunk 1]
```diff
@@ -869,6 +869,14 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n\033[91mUnable to install hostapd from the source!\033[0m\n' +
```
Comment (sophron) [inline wifiphisher.py]: We never tried to install hostapd from the source. The error message should be something like "Unable to install the 'hostapd' package!"

File: wifiphisher.py [hunk 2]
```diff
@@ -869,6 +869,14 @@ def get_hostapd():
...
+            "Please follow the link below to configure your sources.list\n" +
+            "\033[94mhttp://docs.kali.org/general-use/kali-linux-sources-list-repositories\033[0m\n" +
+            "\nThen run apt-get update for changes to take effect!"
```
Comment (sophron) [inline wifiphisher.py]: I believe we should also mention here that Internet connection is necessary to install the package. Something like "Then, while connected to the Internet, run apt-get update for the changes to take effect and rerun the script.".

File: wifiphisher.py [hunk 3]
```diff
@@ -869,6 +869,14 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n\033[91mUnable to install hostapd from the source!\033[0m\n' +
+            "Please follow the link below to configure your sources.list\n" +
+            "\033[94mhttp://docs.kali.org/general-use/kali-linux-sources-list-repositories\033[0m\n" +
```
Comment (sophron) [inline wifiphisher.py]: We have some <a href="https://github.com/HassenPy/wifiphisher/blob/patch-4/wifiphisher.py#L32-L41">constants for the coloring</a>. You should make use of them.

Also, try to follow our convention for error messages.We usually add a `'['+R+'-'+W+']'` before the actual message. Check <a href="https://github.com/HassenPy/wifiphisher/blob/patch-4/wifiphisher.py#L870">here</a> for example.

File: wifiphisher.py (hunk 2 above)
Comment (sophron) [inline wifiphisher.py]: Also, output the `'\n[' + R + '!' + W + '] Closing'` string since we are terminating here.

**Comment by HassenPy** [comment]
new look

File: wifiphisher.py [hunk 4]
```diff
@@ -869,6 +869,16 @@ def get_hostapd():
         else:
             sys.exit(('[' + R + '-' + W + '] hostapd' +
                      'not found in /usr/sbin/hostapd'))
+    if not os.path.isfile('/usr/sbin/hostapd'):
+        sys.exit((
+            '\n[' + R + '-' + W + '] Unable to install the \'hostapd\' package!\n' +
+            '[' + T + '*' + W + '] This process requires a persistent internet connexion!\n' +
```
Comment (sophron) [inline wifiphisher.py]: Typo. It's "connection".

File: wifiphisher.py [hunk 5]
```diff
@@ -869,6 +869,16 @@ def get_hostapd():
...
+            'Please follow the link below to configure your sources.list\n' +
+            B + 'http://docs.kali.org/general-use/kali-linux-sources-list-repositories\n' + W +
+            '[' + G + ' + ' + W + '] Run apt-get update for changes to take effect.\n' +
+            '[' + G + ' + ' + W + '] Rerun the script again to install hostapd.\n' +
```
Comment (sophron) [inline wifiphisher.py]: I think it's better if you remove the spaces that surround the "+" symbol to be similar with the rest of prefixes.

**Comment by HassenPy** [comment]
Dat typo, i red some french documentation this morning haha!

**Comment by sophron** [comment]
It's OK. Thanks for contributing.

**Pull request merged**